timely/src/dataflow/operators/capability.rs

```diff
@@ -478,7 +478,7 @@ impl<T: Timestamp> CapabilitySet<T> {
             // we outline it
             panic!(
                 "Attempted to downgrade a CapabilitySet with a frontier containing elements \
-                which where were not not `less_equal` than all elements within the set"
+                which where were not beyond all elements within the set"
```

Comment:
Nit, but this might also be a good time to change this text. It should "... containing an element that was not beyond an element within the set".